app/src/main/java/org/mifos/mobilebanking/models/Currency.java

```diff
@@ -40,6 +40,9 @@ public void setDecimalPlaces(int decimalPlaces) {
     }
 
     public String getDisplaySymbol() {
+        if (displaySymbol == null) {
+            displaySymbol = getCode();
+        }
```

Comment:
@luckyman20 PR looks good, I want a small change 
```
 public String getDisplaySymbol() {
        if(displaySymbol == null) {
            return code;
        }
        return displaySymbol;
    }
```

Reply:
@therajanmaurya Made the changes